docs/docs/frigate/installation.md

```diff
@@ -5,6 +5,12 @@ title: Installation
 
 Frigate is a Docker container that can be run on any Docker host including as a [HassOS Addon](https://www.home-assistant.io/addons/). Note that a Home Assistant Addon is **not** the same thing as the integration. The [integration](/integrations/home-assistant) is required to integrate Frigate into Home Assistant.
 
+:::tip
+
+If you already have Frigate installed, check out the [getting started guide](../guides/getting_started#configuring-frigate) to configure Frigate.
```

Comment:
```suggestion
If you already have Frigate installed as a Home Assistant addon, check out the [getting started guide](../guides/getting_started#configuring-frigate) to configure Frigate.
```

Reply:
I wasn't sure what "installed" really meant outside of an addon. Figured this statement is mostly targeted at addon users who may not be as technical.